lib/index.js

```diff
@@ -1,23 +1,25 @@
 'use strict';
 
+const crypto = require('crypto');
+const path = require('path');
+const { promisify } = require('util');
+
 const aws = require('aws-sdk');
 const cheerio = require('cheerio');
-const crypto = require('crypto');
+const fs = require('fs-extra');
 const mustache = require('mustache');
-const path = require('path');
 const sass = require('node-sass');
-const verify = require('./verify');
+const tar = require('tar');
+
 const config = require('./config');
 const { logger } = require('./logging');
+const verify = require('./verify');
 
-const { promisify } = require('util');
-const fs = require('fs-extra');
-['readFile', 'writeFile', 'readdir', 'copy'].forEach((fn) => {
+// XXX: Aren't these already promisified with fs-extra?
```

Comment:
You are right. I was so dumb to try `util.promisify` with standard `fs`. Please remove this wrapper and the `const { promisify } = require('util');` above.